azure-quantum/azure/quantum/qiskit/backends/rigetti.py

```diff
@@ -94,6 +94,7 @@ def __init__(self, name: str, provider: "AzureQuantumProvider", **kwargs):
                 "open_pulse": False,
                 "gates": [{"name": "TODO", "parameters": [], "qasm_def": "TODO"}],
                 "azure": self._azure_config(),
+                "is_default": True,
```

Comment:
I think we'll also need to add `is_default` flag onto the `RigettiQPUBackend`. As far as I see, we use `is_default` flag to deduplicate backends having the same Target name. Since we're now going to return only the default backends from `provider.backends()` we should make sure that:
- If backend has unique target name - we set `is_default=True` on it.
- If there are more than one backend with the same target name - we set `is_default=True` on only one of them.
Could you please make sure that one of the above is done for each Qiskit backend?